domain/schema/model/sql/0025-model-status.sql

```diff
@@ -8,8 +8,7 @@ INSERT INTO model_status_value VALUES
 (0, 'error'),
 (1, 'available'),
 -- We set the model status to busy when the model is being upgraded.
-(2, 'busy'),
-(3, 'suspended');
+(2, 'busy');
```

Comment:
We are going to derive this from whether a model is migrating, right?

Reply:
We still use busy for `upgrade`.